framework/source/class/qx/ui/table/model/Filtered.js

```diff
@@ -79,10 +79,10 @@ qx.Class.define("qx.ui.table.model.Filtered",
   {
     this.base(arguments);
 
-    this.numericAllowed = new Array("==", "!=", ">", "<", ">=", "<=");
-    this.betweenAllowed = new Array("between", "!between");
+    this.numericAllowed = ["==", "!=", ">", "<", ">=", "<="];
```

Comment:
writing
```javascript
this.filterType = {
   "==": 'numeric',
   ....
   "between": 'between'
};

...

and then instead of the `qx.lang.Array.contains`

```javascript
this.filterType[filter] == 'between';
```

or even

```javascript
switch(this.filterType[filter]){
'between': ...
   break;
...
}

```

or even

```javascript
switch(